demo/index.css

```diff
@@ -1,46 +1,579 @@
-﻿[draggable] {
-  user-select: none;
+﻿* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Poppins, sans-serif;
+    font-weight: 400;
+}
+
+html,
+body {
+    position: relative;
+    width: 100%;
+    height: 100dvh;
+    overflow: hidden;
+}
+
+h1,
+h2,
+h3 {
+    letter-spacing: -0.025em;
+    margin-bottom: 1.25em;
+
+    a {
+        color: black;
+        text-decoration: none;
```

Comment:
Let's not mess with accessibility: this is not an app, it's still a normal webpage, so links can stay default coloured and underlined.